README.md

```diff
@@ -1 +1,55 @@
-# Hello_World
\ No newline at end of file
+# Project name
+HelloWorld
```

Comment:
Rather than using the title `Project name` you can change that to your project name which is `Hello Microverse`.

Reply:
changed to Hello Microverse as advised